### DISCUSSION TITLE
[Bug] Stop hooking Ctrl+Shift+K for "Run A Command"

### DISCUSSION THREAD

**Discussion by AliveDevil** [community/community#7904] [Other Feature Feedback, Questions, & Ideas] [answered]
On Microsoft Edge Ctrl+Shift+K is used for duplicate tab.
"Run a command" hooking into Ctrl+Shift+K breaks my workflow of duplicating a tab fast.

As of [your own documentation](https://docs.github.com/en/get-started/using-github/github-command-palette) this should only happen with Ctrl+K, not Ctrl+Shift+K.

**Comment by lnlp** [comment]
Not sure if this was recently introduced in GitHub (apparently it was) but GitHub currently hijacks/breaks the Ctrl+Shift+K (duplicate current tab) behavior in Microsoft Edge which is annoying.

Hope this will get fixed soon.

_Maybe 'eat your own (Microsoft) dogfood' will help prevent issues like this?_

**Comment by AliveDevil** [comment, marked as answer]
Don't know what to make out of it, that since months there hasn't been a conclusive decision. Disabling command palette then is the solution.

* https://github.community/t/conflicts-with-browser-shortcut/208860
* https://github.community/t/ctrl-k-is-for-jumping-to-the-firefox-searchbar-are-you-drunk/209410

**Reply by lnlp** [reply]
Disabling command palette is not really a solution. 
It's annoying that this poorly implemented feature has still not been fixed and one needs to manually disable it to stop hi-jacking of the duplicate tab key stroke.

**Comment by cobrexus** [comment]


**Reply by AliveDevil** [reply]
Functionality is different. Duplicate Tab puts the tab right next to the current tab while `Alt-D, Alt-Enter` puts it to the right-most location.

**Reply by Alex9001** [reply]
ctrk+k goes to the search bar, alt+d to the address bar. Both will allow you to perform a search but only the search bar will save form history. The workaround is creating/logging in to an account and disabling shortcut in accessibility settings.

**Comment by gaknoll** [comment]
Hi all! We appreciate your patience on being able to customize your command palette keyboard shortcuts. Today we [shipped a new feature](https://github.blog/changelog/2022-02-03-customizing-command-palette-keyboard-shortcuts-beta/) that allows you to customize the keyboard shortcuts used to open the command palette in search and command modes. Update your preferences in the [Accessibility section](https://github.com/settings/accessibility) of your user settings.
